packages/sources/anchorage/src/transport/wallet.ts

```diff
@@ -0,0 +1,162 @@
+import { TransportDependencies } from '@chainlink/external-adapter-framework/transports'
+import { ResponseCache } from '@chainlink/external-adapter-framework/cache/response'
+import { Requester } from '@chainlink/external-adapter-framework/util/requester'
+import { AdapterResponse, makeLogger, sleep } from '@chainlink/external-adapter-framework/util'
+import { SubscriptionTransport } from '@chainlink/external-adapter-framework/transports/abstract/subscription'
+import { EndpointContext } from '@chainlink/external-adapter-framework/adapter'
+import { BaseEndpointTypes, inputParameters } from '../endpoint/wallet'
+import { AdapterError } from '@chainlink/external-adapter-framework/validation/error'
+
+const logger = makeLogger('WalletTransport')
```

Comment:
Use this name needs to be unique? I see another WalletTransport exist already

Reply:
Not at all. On adapter level this should be unique between transports, but since there is one transport it can be anything.